tests/test_services/test_llm_service_ollama.py

```
@@ -0,0 +1,108 @@
+"""
+LLMService の Ollama Cloud 経路テスト（Phase A）
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)